packages/openapi-specification/openapi.yaml

```diff
@@ -1,742 +1,177 @@
+      operationId: addRecord
+      requestBody:
+        $ref: '#/components/requestBodies/Record'
       responses:
         '200':
-          description: successful operation
+          description: Success
           content:
-            application/xml:
-              schema:
-                $ref: '#/components/schemas/Pet'
             application/json:
               schema:
-                $ref: '#/components/schemas/Pet'
+                $ref: '#/components/schemas/Record'
         '400':
-          description: Invalid ID supplied
-        '404':
-          description: Pet not found
-      security:
-        - api_key: []
-    post:
-      tags:
-        - pet
-      summary: Updates a pet in the store with form data
-      description: ''
-      operationId: updatePetWithForm
-      parameters:
-        - name: petId
-          in: path
-          description: ID of pet that needs to be updated
-          required: true
-          schema:
-            type: integer
-            format: int64
-      responses:
-        '405':
           description: Invalid input
       security:
-        - petstore_auth:
-            - 'write:pets'
-            - 'read:pets'
-      requestBody:
-        content:
-          application/x-www-form-urlencoded:
-            schema:
-              type: object
-              properties:
-                name:
-                  description: Updated name of the pet
-                  type: string
-                status:
-                  description: Updated status of the pet
-                  type: string
-    delete:
-      tags:
-        - pet
-      summary: Deletes a pet
-      description: ''
-      operationId: deletePet
-      parameters:
-        - name: api_key
-          in: header
-          required: false
-          schema:
-            type: string
-        - name: petId
-          in: path
-          description: Pet id to delete
-          required: true
-          schema:
-            type: integer
-            format: int64
-      responses:
-        '400':
-          description: Invalid pet value
-      security:
-        - petstore_auth:
-            - 'write:pets'
-            - 'read:pets'
-  '/pet/{petId}/uploadImage':
-    post:
+        - records_auth:
+          - write:records
+          - read:records
+
+    put:
       tags:
-        - pet
-      summary: uploads an image
-      description: ''
-      operationId: uploadFile
+        - website-record
+      summary: Update an existing website record by ID
+      description: Updates one of the records in the list of website records.
+      operationId: updateRecord
       parameters:
-        - name: petId
-          in: path
-          description: ID of pet to update
+        - name: recordId
+          in: query
+          description: The ID of the record that should be modified
           required: true
           schema:
             type: integer
             format: int64
```

Comment:
Do we want to use this as an ID or string of format UUID?

Reply:
I put it as int because in our SQL database with all the records, the IDs will be integers, no?

Reply:
We can actually put the primary key constraint to an arbitrary column, but I don't mind the integers... easier to look at I guess 😄